Makefile

```diff
@@ -98,7 +98,12 @@ out/Makefile: common.gypi deps/uv/uv.gyp deps/http_parser/http_parser.gyp \
 	$(PYTHON) tools/gyp_node.py -f make
 
 config.gypi: configure
-	$(error Missing or stale $@, please run ./$<)
+	@if [ -x config.status ]; then \
+		./config.status; \
```

Comment:
I'd rather have:
```make
$(PYTHON) configure $(something that reads .configure_args as args)
```
So it will be easier to implement for Windows

Reply:
btw shouldn't this be dependant on `ls -R *.gyp?` as well?

Reply:
> So it will be easier to implement for Windows

We could follow the same approach pretty easily, I think

> btw shouldn't this be dependant on `ls -R *.gyp?` as well?

I don’t know, somebody else would have to answer that

Reply:
Now I understand why your pattern makes perfect sense for autotools, since it only assumes a POSIX compatible shell...

Reply:
Yeah … I heard escaping parameters for cmd is fun :)

Otherwise we should be able to follow the same format here if we like

Reply:
> shouldn't this be dependant on `ls -R *.gyp`?

Ideally, yes, but can be done in a separate PR.